docsets/NetworkX/docset.json

```diff
@@ -1,15 +1,21 @@
 {
     "name": "NetworkX",
-    "version": "1.11",
-    "archive": "NetworkX.tgz",
+    "version": "2.1",
+    "archive": "versions/2.1/NetworkX.tgz",
```

Comment:
This is not currently supported. The archive needs to be different from the `specific_versions` ones.

Reply:
Should there be a version 2.1 in both the NetoworkX-folder and in a version-folder?

Reply:
The archive is a duplicate, yes.